storm-client/src/jvm/org/apache/storm/task/IOutputCollector.java

```diff
@@ -30,4 +31,5 @@
     void ack(Tuple input);
     void fail(Tuple input);
     void resetTimeout(Tuple input);
+    void flush();
```

Comment:
(Random passer-by user: E.g. we do this for tests a lot, yes. On the other hand, this is for Storm 2.0, so API breakages are kinda expected. Then again, at least it should be thinked through / discussed.)